guides/common/modules/con_defining-content-access-strategies-for-hosts.adoc

```diff
@@ -0,0 +1,106 @@
+You can add content overrides to hosts or activation keys.
++
+[NOTE]
+====
+By default, content hosted by {Project} can be either enabled or disabled.
+When a repository is enabled, hosts can access the repository packages or other content, allowing hosts to download and install the available content.
+When a repository is disabled, the host is not able to access the repository content.
+
+ifdef::orcharhino[]
+ifdef::red_hat_enterprise_linux[]
+In custom products, repositories are always disabled by default, while Red{nbsp}Hat products can be either enabled or disabled by default depending on the specific repository.
+endif::[]
+ifndef::red_hat_enterprise_linux[]
+In custom products, repositories are always disabled by default.
+endif::[]
+endif::[]
+ifndef::orcharhino[]
+In custom products, repositories are always disabled by default, while Red{nbsp}Hat products can be either enabled or disabled by default depending on the specific repository.
+endif::[]
+====
++
+* For more information, see {ManagingHostsDocURL}Enabling_and_Disabling_Repositories_on_Hosts_managing-hosts[Enabling and Disabling Repositories on Hosts] in _{ManagingHostsDocTitle}_ and {ContentManagementDocURL}enabling-and-disabling-repositories-on-activation-key_content-management[Enabling and disabling repositories on activation key] in _{ContentManagementDocTitle}_.
+. _Define composite content views_. You can use composite content views to combine and give hosts access to the content from multiple content views.
++
+For more information, see link:{ContentManagementDocURL}Creating_a_Composite_Content_View_content-management[Creating a composite content view] in _{ContentManagementDocTitle}_.
+. _Define architecture and operating system version restrictions._
+In custom products, you can set restrictions on the architecture and operating system versions for `{client-content-type}` repositories on which the product will be available.
+For example, if you restrict a custom repository to *{client-os} {client-os-major}*, it is only available on hosts running {client-os} {client-os-major}.
++
+ifdef::orcharhino[]
+ifdef::red_hat_enterprise_linux[]
+Red{nbsp}Hat repositories set architecture and operating system version restrictions automatically.
+endif::[]
+endif::[]
+ifndef::orcharhino[]
+Red{nbsp}Hat repositories set architecture and operating system version restrictions automatically.
+endif::[]
++
+[IMPORTANT]
+====
+Architecture and operating system version restrictions hold the highest priority among all other strategies.
+They cannot be overridden or invalidated by content overrides, changes to content views, or changes to lifecycle environments.
+For this reason, consider the other strategies before implementing architecture or operating system version restrictions.
+====
+. _Define release version restrictions._
+ifndef::orcharhino[]
+Certain Red{nbsp}Hat repositories, such as the {RHEL} dot release repositories, include a *Release version* in their repository metadata.
```

Comment:
```suggestion
Certain Red{nbsp}Hat repositories, such as the {RHEL} minor release repositories, include a *Release version* in their repository metadata.
```
Not sure what the "dot" means :)

Reply:
Eh, me neither. I was hoping to clarify it later, your suggestion works, thanks!